README.md

```diff
@@ -365,7 +373,25 @@ maintenance and customization.
 
  * `Kuiper with exported sources:` 
 
-   - **07.export-stage** - substage **02.export-sources**
+   - **08.export-stage** - substage **02.export-sources**
+
+ * `Kuiper with custom script:`
+
+   - **07.extra-scripts**
+
+# Kuiper with custom extra scripts
+
+Kuiper has an option to run extra scripts at build time in order to customize the resulted image.
+
+Steps to prepare Kuiper customization with the example script:
+- in the `config` file set 'EXTRA_SCRIPT' variable to `stages/07.extra-scripts/examples/extra-script-example.sh`
+- if you need to pass `config` file parameters to the script uncomment the version with argument passing in `stages/07.extra-scripts/run.sh`, otherwise skip this step
```

Comment:
Can you please reformulate a bit ? or maybe just add missing commas will be enough :)